audit/projects/kubernetes-public/buckets/kubernetes-public_cloudbuild/bucketpolicyonly.txt

```diff
@@ -0,0 +1,3 @@
+Bucket Policy Only setting for gs://kubernetes-public_cloudbuild:
```

Comment:
Are we using kubernetes-public as a project for cloudbuild? This seems unexpected to me (this applies to all the files in this dir)

Reply:
That was also confusing for me but I didn't do research about it yet

Reply:
@dims your fingerprints are on this - April 20, 2020 at 5:47:58 PM GMT-7

Can you explain this?  It looks to be something "gcb-docker-gcloud" ?  I suggest we nuke this and be aware of how it came to be

Reply:
Not sure what exactly i did here ... i was working on testing the `gcb-docker-gcloud` image using https://cs.k8s.io/?q=gcb-docker-gcloud&i=nope&files=&repos= and probably forgot to switch from community credentials (davanum@gmail.com) to my office one (dims@vmware.com)

Reply:
OK. I will nuke it